### DISCUSSION TITLE
Native Rocker Gestures

### DISCUSSION THREAD

**Discussion by devployment** [zen-browser/desktop#1740] [Ideas]
Having "Rocker Gestures" built in, like Vivaldi has would be great.

https://help.vivaldi.com/desktop/shortcuts/rocker-gestures/

**Comment by LogicDaemon** [comment]
I agree, for me it's the main showstopper for actually trying Zen currently

**Comment by LogicDaemon** [comment]
@devployment btw, you say "Built in"… are you implying there is a working addon or extension which adds that functionality? I've searched one for Firefox or Chrome low and high until Vivaldi got the feature (and then I switched to Vivaldi immediately despite it didn't even have the Sync yet).

**Reply by devployment** [reply]
Cannot remember what I used exactly. But there are a ton of Mouse gesture plugins available.

**Reply by devployment** [reply]
The thing is that I want to minimize the amount of plugins to use. Vivaldi does a great job at this as it adds already plenty of features out of the box.

**Comment by LogicDaemon** [comment]
yea there are tons of mouse plugins and I tried a half of them. Some even promised the rocker gesture, but it never worked like in the old Opera/Vivaldi (when I can hold RMB and click LMB multiple times). I guess, that's because they try to capture the input in the document context, but switching pages causes reload.

**Comment by devployment** [comment]
Just for my curiosity. If someone might want to contribute to this, what would be a good place to start looking at? 

Is it even feasible to attempt something like this?

**Comment by lincolnep** [comment]
I would love this feature added to Zen as it would give a better feel and work for all tabs.
Currently I am using the plugin Gesturefy which works for most tabs just the settings ones do not work
In Gesturefy Rocker settings are under Extras
